eventstore/management/commands/migrate_to_eventstore.py

```diff
@@ -266,6 +268,40 @@ def handle_languageswitch_change(self, change):
             },
         )
 
+    def handle_eddswitch_change(self, change):
+        data = change.data or {}
+        new_edd = data.pop("edd")
+        old_edd = data.pop("old_edd") or ""
+        EddSwitch.objects.update_or_create(
+            id=change.id,
+            defaults={
+                "contact_id": change.registrant_id,
+                "source": change.source.name,
+                "old_edd": old_edd,
+                "new_edd": new_edd,
+                "timestamp": change.created_at,
+                "created_by": change.source.user.username,
+                "data": data,
+            },
+        )
+
+    def handle_babydobswitch_change(self, change):
+        data = change.data or {}
+        new_baby_dob = data.pop("baby_dob")
+        old_baby_dob = data.pop("old_baby_dob") or ""
+        BabyDobSwitch.objects.update_or_create(
+            id=change.id,
+            defaults={
+                "contact_id": change.registrant_id,
+                "source": change.source.name,
+                "old_baby_dob": old_baby_dob,
+                "new_baby_dob": new_baby_dob,
+                "timestamp": change.created_at,
+                "created_by": change.source.user.username,
+                "data": data,
+            },
+        )
```

Comment:
EDD and Baby DoB changes are not something that we currently allow on Seed, so we don't need to migrate them. If there was something for Seed that we needed to migrate here, we would need to add it to the `handle` function, otherwise these functions will never be called.